src/tree/SubscriptionTreeItem.ts

```diff
@@ -104,12 +103,15 @@ export class SubscriptionTreeItem extends SubscriptionTreeItemBase {
         }
 
         if (!context.advancedCreation) {
+            LocationListStep.addStep(wizardContext, promptSteps);
```

Comment:
Since custom locations are only supported for advanced scenarios, would it be worth it to leave location list step at the end for basic create? That way people using the more common scenario won't notice a change.

Reply:
It's still at the end